signingworker/data/signing_task_schema.json

```diff
@@ -0,0 +1,24 @@
+{
+    "title": "Taskcluster MAR signing task minimal schema",
+    "type": "object",
+    "properties": {
+        "scopes": {
+            "type": "array",
+            "minItems": 1,
+            "uniqueItems": true,
+            "items": {
+                "type": "string"
+            }
+        },
+        "payload": {
+            "type": "object",
+            "properties": {
+                "signingManifest": {
+                    "type": "string"
+                }
+            },
```

Comment:
As I mentioned in person, the idea is to move manifest's content to the payload after we have some stable format. It's just easier to manipulate the manifest without changing tasks.